File: app/src/main/java/bobrchess/of/by/belaruschess/presenter/callback/CallBackTournament.java
package bobrchess.of.by.belaruschess.presenter.callback;

import java.util.List;

import bobrchess.of.by.belaruschess.dto.GameDTO;
import bobrchess.of.by.belaruschess.dto.TournamentDTO;

/**
 * Created by Igor on 11.04.2018.
 */

public interface CallBackTournament extends CallBack {
    void onResponse(TournamentDTO tournamentDTO);

    void onResponse(List list);
}

File: app/src/main/java/bobrchess/of/by/belaruschess/presenter/callback/CallBackTournament.kt
package bobrchess.of.by.belaruschess.presenter.callback

import bobrchess.of.by.belaruschess.dto.TournamentDTO

interface CallBackTournament : CallBack {
    fun onResponse(tournamentDTO: TournamentDTO)

    fun onResponse(list: List<*>)
}
